addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py

```diff
@@ -109,6 +109,10 @@ def __gather_double_sided(blender_material, mesh_double_sided, export_settings):
 
 
 def __gather_emissive_factor(blender_material, export_settings):
+    # An Emission node, alone, should be considered an Unlit material.
+    if not __has_node_of_type(blender_material, bpy.types.ShaderNodeBsdfPrincipled):
```

Comment:
If the legacy custom group node is in use, this test will give the wrong answer.